src/util/Well.tsx

```diff
@@ -54,9 +56,7 @@ const Well: FunctionComponent<IWellProps> = (props) => {
       style={style}
     >
       <div style={{ width: '100%' }}>
-        {props.text !== undefined
-          ? props.text
-          : props.wellPlate.getPosition(props.value, 'formatted')}
+        {displayableValue === undefined ? null : displayableValue}
```

Comment:
this seems wrong? if the value returned by `props.text` is `undefined`, we should still display the position by default.